### DISCUSSION TITLE
Uno Platform - Non flat 3D controls.

### DISCUSSION THREAD

**Discussion by ward-horsfall** [unoplatform/uno#16225] [Q&A]
Hi,

It seems the current trend is flat non 3D controls. 
What I am looking for is something that was "Vista like"
colorfull and 3D in how it did its UI.

Is there any free or commercial pre-made options for this?

If there is not how hard is it to do? I assume a lot of work.

Would this come by way of another toolkit.

If there is not how hard is it to do? (I assume a lot of work).

Where would a starting point be in the UNO documentation.

Thanks,

Ward

**Comment by francoistanguay** [comment]
Do you mean 2D controls that have a 3D feel to them?

If so, you can go look the Toolkit/ShadowContainer section of our gallery https://gallery.platform.uno/

As well as our docs: https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/controls/ShadowContainer.html#neumorphism

![image](https://github.com/unoplatform/uno/assets/30851132/03bd9cf6-c721-4fc5-9505-86cb331b2bb0)
